### ISSUE TITLE
Extract the image_list.json creation to a parallel CI job

### ISSUE THREAD

**Issue by jimmykarily** [cloudfoundry-incubator/kubecf#865]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**
The task that creates the image_list.json for the kubecf bundle can take up to 25 minutes to complete (https://concourse.suse.dev/teams/main/pipelines/kubecf/jobs/build-master/builds/58). That file is not needed during any of the tests we run in CI thus can be generated in parallel without blocking the next jobs.

We should investigate if it's possible to have a separate (parallel) job that generates that file and injects it in the chart tarball after all tests have passed and before the publish step (or in the publish step itself).

**Comment by jandubois** [comment]
Since #965 creation of the imagelist takes only seconds (because it only evaluates 2 different combinations of feature flags), so I think this issue is no longer relevant.
